components/brave_rewards/resources/shared/lib/onboarding_completed_store.ts

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// An OnboardingCompletedStore can be used by apps that include
+// rewards onboarding UX to store whether the user has interacted
+// with the onboarding experience.
+export class OnboardingCompletedStore {
+
+  load () {
+    return Boolean(localStorage.rewardsOnboardingComplete)
+  }
+
+  save () {
+    localStorage.rewardsOnboardingComplete = String(Date.now())
```

Comment:
totally minor, just curious. you save the actual timestamp in `save` while returning a boolean from `load` -- do you see a need to expose the timestamp in the future?

Reply:
I generally don't like to "program ahead", but I thought it might be useful in the future to be able to tell when a user last interacted with the onboarding UX (at the given entry point) so that we'd be able to do things like: if the user hasn't dismissed us in the last X days on the panel, show a reminder CTA (and we'd already have that info in local storage).